commit 825ec0d3b3dea786d85f11eadf831051f611ef77 (refs/changes/51/2551/4)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-09-25T20:32:11+00:00 (2 years ago)
    
    MORAY-432 test/client.test.js sporadically fails
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/test/client.test.js b/test/client.test.js
index 37dae22..2631780 100644
--- a/test/client.test.js
+++ b/test/client.test.js
@@ -91,7 +91,7 @@ test('close() with outstanding callback request', function (t) {
             t.end();
         });
 
-        setImmediate(function () { c2.close(); });
+        c2.close();
     });
 });
 
@@ -118,7 +118,7 @@ test('close() with outstanding event-emitter request', function (t) {
             t.end();
         });
 
-        setImmediate(function () { c2.close(); });
+        c2.close();
     });
 });
 
